oak/server/rust/oak_runtime/src/metrics/mod.rs

```diff
@@ -103,18 +125,92 @@ impl GrpcServerMetrics {
                 &["method_name"],
                 "Histogram of response latency of RPCs handled by the server.",
             )),
-            grpc_response_size_bytes: builder.register(histogram_vec(
-                "grpc_response_size_bytes",
+            grpc_server_response_size_bytes: builder.register(histogram_vec(
+                "grpc_server_response_size_bytes",
                 &["method_name"],
                 "Histogram of response sizes of RPCs handled by the server.",
             )),
             grpc_server_msg_sent_total: builder.register(histogram_vec(
```

Comment:
What is this histogram bucketed by?

Reply:
This is now fixed in a separate PR: #1228